src/mode/javascript.js

```diff
@@ -10,9 +10,16 @@ var CStyleFoldMode = require("./folding/cstyle").FoldMode;
 
 var Mode = function() {
     this.HighlightRules = JavaScriptHighlightRules;
-    
+
     this.$outdent = new MatchingBraceOutdent();
-    this.$behaviour = new CstyleBehaviour();
+    this.$behaviour = new CstyleBehaviour({
+        quotesPrefixes: [
+            {
```

Comment:
This configuration is too complex, and too different from the configuration of quotes. Could we use something like `mode.pairQuotesAfter: {"`": /\w/}`.

Reply:
Done